accelerator/errors.h

```diff
@@ -195,6 +196,19 @@ typedef enum {
   /**< Error in setting options of `struct mosquitto` object */
   SC_CLIENT_CONNTECT = 0x0A | SC_MODULE_MQTT | SC_SEVERITY_MAJOR,
   /**< Error in connecting to broker */
+
+  // STORAGE module
+  SC_STORAGE_OOM = 0x01 | SC_MODULE_STORAGE | SC_SEVERITY_FATAL,
+  /**< Fail to malloc space for transactions */
+  SC_STORAGE_CONNECT_FAIL = 0x02 | SC_MODULE_STORAGE | SC_SEVERITY_MAJOR,
+  /**< Fail to connect scylla node */
+  SC_STORAGE_INVAILD_INPUT = 0x03 | SC_MODULE_STORAGE | SC_SEVERITY_MAJOR,
+  /**< invaild input parameter, e.g., null pointer or   */
+  SC_STORAGE_CASS_ERROR = 0x04 | SC_MODULE_STORAGE | SC_SEVERITY_MAJOR,
```

Comment:
`SC_STORAGE_CASS_ERROR` might be not mnemonic enough. While executing Cassandra APIs, did you perform queries all the time?

Reply:
`CASS` here is misleading to me.